src/plugins/uploader.js

```diff
@@ -0,0 +1,152 @@
+"use strict";
+
+const SocketIOFileUploadServer = require("socketio-file-upload/server");
+const Helper = require("../helper");
+const path = require("path");
+const fsextra = require("fs-extra");
+const fs = require("fs");
+const fileType = require("file-type");
+const readChunk = require("read-chunk");
+const crypto = require("crypto");
+const isUtf8 = require("is-utf8");
+const log = require("../log");
+
+const whitelist = [
+	"application/ogg",
+	"audio/midi",
+	"audio/mpeg",
+	"audio/ogg",
+	"audio/x-wav",
+	"image/bmp",
+	"image/gif",
+	"image/jpeg",
+	"image/png",
+	"image/webp",
+	"text/plain",
```

Comment:
Just curious, things like log files (often in `.log`) and Markdown files (`.md`), they would be  treated as plain text, right? (I haven't tested yet, doing code review first then I'll play with the feature)

Reply:
The extension doesn't matter, it detects the type from the file contents, so yes logs or markdown will work fine.

Reply:
Awesome, that's what I thought, thanks for confirming!